app/response_processor.py

```diff
@@ -250,11 +249,11 @@ def send_receipt(self, decrypted_json):
             self.logger.exception("Unsuccesful publish")
             raise RetryableError
 
-    def send_notification(self):
+    def send_notification(self, id_tag):
         self.logger.info("Sending to downstream")
         try:
             self.logger.info("About to publish notification to queue")
-            self.notifications.publish_message(self.tx_id, headers={'tx_id': self.tx_id})
+            self.notifications.publish_message(json.dumps(id_tag), headers={'tx_id': self.tx_id})
```

Comment:
I think rather than change the payload of the message we should add a header to send to downstream with an indication whether its a feedback message. 

e.g. we would do this

headers={'tx_id': self.tx_id, 'feedback': 'true;})

Reply:
as discussed this comment can be ignored now